src/sign.ts

```diff
@@ -98,12 +90,21 @@ export class Signer {
       `https://rekor.sigstore.dev/api/v1/log/entries/${entry.uuid}`
     );
 
-    const signedPayload: SignedPayload = {
-      base64Signature: signature,
+    const bundle: SigstoreBundle = {
+      attestationType: 'attestation/blob',
+      attestation: {
+        payloadHash: digest,
+        payloadAlgorithm: 'sha256',
+        base64Signature: signature,
```

Comment:
Looking at the [current proposal](https://docs.google.com/document/d/1gucjOA_bGyRjK6TeaOI-X5GIUv8WsPzeMDMkq25Kv4Y/edit#heading=h.yv4l74tld8up)

```suggestion
        payloadHashAlgorithm: 'sha256',
        signature: signature,
```